lethal-squirrels/workforce-management
# Computers work asshole

## REFERENCE

Ticket:  (#3)

## DESCRIPTION
Updated computer information in the employee detail view. 

## STEPS TO TEST
1. Clone this branch to a local repo
2. In the package manager, run `update-database`. 
3. Test functionality
  * press `ctrl + fn + f5` 
  * navigate to `localhost/employees` and click on an employee detail
  * make sure a computer shows up
